R/aes_labs.R

```diff
@@ -0,0 +1,545 @@
+#' Get aesthetic titles, subtitles and footnotes.
+#' plot - Plot object to adjust.
+#' plot_width - Numeric. Width of the plot.
+#' @noRd
+update_labs <- function(plot, plot_width){
+
+  p <- ggplot2::ggplotGrob(plot)
```

Comment:
I believe that you do not need to call the `ggplot2::` namespace explicitly in theme61 because we have imported all the package functions (see the `import(ggplot2)` line in the NAMESPACE file).

I think we did this to save on typing ggplot2 every time, since this package would use functions from there all the time.

Obviously if we are calling a function that we have masked then we need to name the namespace explicitly.

Also noting you're tacitly aware of this since there's a few ggplot functions that have not been namespaced.

Reply:
Yeah good point. I think I went through and added the namespace calls to all non-base functions at one stage and then have been a bit sloppy about updating it. But happy to do a remove all the ggplot calls (I assume you have already done this)

Reply:
Yep